src/app/core/services/label.service.ts

```diff
@@ -6,6 +6,10 @@ import { Observable } from 'rxjs';
 import { SEVERITY_ORDER } from '../../core/models/issue.model';
 import { User } from '../models/user.model';
 
+const COLOR_DARKNESS_THRESHOLD = 0.199; // The threshold to decide if color is dark or light
```

Comment:
It might be helpful for future devs if you mention the acceptable range. i.e., max and min values